src/main/kotlin/org/kamiblue/client/module/modules/render/NewChunks.kt

```diff
@@ -33,6 +37,10 @@ internal object NewChunks : Module(
     category = Category.RENDER
 ) {
     private val relative by setting("Relative", false, description = "Renders the chunks at relative Y level to player")
+    private val renderMode = setting("RenderMode", RenderMode.BOTH)
+    private val chunkGridColor by setting("Radar Chunk grid color", ColorHolder(255, 0, 0, 100), true, { renderMode.value != RenderMode.WORLD })
+    private val distantChunkColor by setting("Radar Distant chunk color", ColorHolder(100, 100, 100, 100), true, { renderMode.value != RenderMode.WORLD }, "Chunks that are not in render distance and not in baritone cache")
+    private val newChunkColor by setting("Radar New chunk color", ColorHolder(255, 0, 0, 100), true, { renderMode.value != RenderMode.WORLD })
```

Comment:
Change these to `Grid Color`, `Distant Chunk Color` and `New Chunk Color`, respectively